ci/jenkins/jobs/macros.yaml

```diff
@@ -185,14 +187,15 @@
           echo ====== Delivering Antrea to all the Nodes ======
           export KUBECONFIG="${WORKSPACE}/jenkins/out/kubeconfig"
 
-          docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
+          docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:$(DOCKER_IMG_VERSION)
```

Comment:
how / when is `DOCKER_IMG_VERSION` set?

Reply:
Thanks for reminding me. Now I set VERSION in make with "JOB_ID-BUILD_NUMBER" and DOCKER_IMG_VERSION equals to this as well.